Dockerfile

```diff
@@ -58,15 +61,12 @@ RUN (cd /lib/systemd/system/sysinit.target.wants && for i in *; do [ $i == syste
 ## Remove any existing configurations
 RUN rm -f /etc/httpd/conf.d/*
 
-COPY docker-assets/entrypoint /usr/bin
-
 EXPOSE 80
 
 WORKDIR /etc/httpd
 
-RUN systemctl enable dbus httpd
+RUN systemctl enable httpd
 
-VOLUME [ "/sys/fs/cgroup" ]
+VOLUME /tmp /run /var/log/httpd
 
-ENTRYPOINT [ "entrypoint" ]
```

Comment:
@abellotti I need to test, I think oci hooks would be happy as long as ENTRYPOINT is not set to something else, we could potentially hijack the CMD, perform work and then exec the real /sbin/init. Another option is to create an initialization systemd service unit that performs the pre-work and then attempts to bring up httpd. Like we use to have before with "evminit" and "appliance-initialize" units.